.github/CONTRIBUTING.md

```diff
@@ -67,6 +67,37 @@ In addition to our automatically enforced [lint rules](https://github.com/bpmn-i
 
 __Rationale:__ People should be able to consume parts of the library with an ES module aware bundler such as [Webpack](https://webpack.js.org/) or [Rollup](https://rollupjs.org) without the need to use a transpiler such as [Babel](https://babeljs.io/).
 
+### Unit Tests
+
+In order to retrieve a sign-off for your contribution, it needs to be sufficiently and well tested. Please structure your unit tests into [**given**, **when** and **then**](https://gist.github.com/nikku/1dc2edb553565238c7bf). To increase overall readability and understandability please also leave two empty lines before each `describe(...)` and `it(...)` block unless it is the first statement in the current indentation level. Also see this example:
+
+```
```

Comment:
Rather than quoting long lines of tests, maybe it makes sense to link into the existing code base (which is fairly well tested in a consistent manner). 

If we wanted to explain testing in detail here (which we don't, I guess) we'd need to cover so many more things.

Reply:
Good idea, will work on it